README.md

```diff
@@ -106,6 +106,36 @@ You can theme the borders. Here is an card example.
 
 ![Example with card edges](/images/card.png)
 
+Reasons
+----
+
+**Why use &lt;twitter-status&gt; instead of Twitter's embedded tweets?**
+
+- Open source - If you don't like something about it you can customize it to fit your exact needs.
+- Lightweight
+  - A minimal &lt;twitter-status&gt; [example](https://twitter-e9454.firebaseapp.com/twitter-status) comes in at **33KB** before compression. ([With more to save](https://github.com/abraham/twitter-status/issues/79))
+  - A minimal [Twitter Embed](https://developer.twitter.com/en/docs/twitter-for-websites/embedded-tweets/overview.html) [example](https://twitter-e9454.firebaseapp.com/oembed) comes in at **121KB** before compression.
+- Security - You can perform a security audit of &lt;twitter-status&gt; and know exactly what you are shipping. You don't have to worry about loading Twitter's JavaScript.
+- Privacy - &lt;twitter-status&gt; only loads embedded images and videos from Twitter's CDN. You don't have to worry about loading Twitter's JavaScript.
+- Native web component - &lt;twitter-status&gt; is built with standardized web APIs that will work out of the box with [most frameworks](https://custom-elements-everywhere.com/).
+- No framework dependancies - Because it's based on native web components, it does not have a dependency on Angular, React, or any other framework.
+- Custom URL handling ([coming soon](https://github.com/abraham/twitter-status/issues/48)) - When a user click on a #hashtag, @mention, etc, you can configure it that they stay within your site.
+- Cached data - If you are a news organization or displaying tweets from politicians, you can continue displaying deleted tweets.
```

Comment:
Technically true, but this would be against [Twitter's policy](https://developer.twitter.com/en/docs/tweets/filter-realtime/guides/streaming-message-types): 

> Client code must honor [status deletion notices] by clearing the referenced Tweet from memory and any storage or archive

Reply:
That policy is specifically the streaming API. I think most apps don't use that. The general policy is different.

> you will make all reasonable efforts to delete or modify such Twitter Content (as applicable) as soon as reasonably possible, and in any case within 24 hours after a request to do so by Twitter or by a Twitter user

There is [existing precedent](https://www.wired.com/2016/01/twitter-reverses-itself-on-politicians-deleted-tweets/) that politicians tweets are exempt and news organizations regularly display screenshots of tweets for the explicit purpose of maintaining availability even in the case of user deletion.